pkgs/data/fonts/commit-mono/default.nix

```diff
@@ -20,7 +20,8 @@ stdenvNoCC.mkDerivation rec {
 
   installPhase = ''
     runHook preInstall
-    install -Dm644 -t $out/share/fonts/opentype/ *.otf
+    install -Dm644 -t $out/share/fonts/opentype/ CommitMono-${version}/*.otf
```

Comment:
Why is the version needed here?

Reply:
this seems to be the structure of upstream's zip file

<details>
<summary>Details</summary>

```
> ouch l CommitMono-1.138.zip --tree
Archive: ~/CommitMono-1.138.zip
└── CommitMono-1.138
   ├── CommitMono-400-Italic.otf
   ├── custom-settings.json
   ├── CommitMono-700-Regular.otf
   ├── CommitMono-400-Regular.otf
   ├── ttfautohint
   │  ├── CommitMono-400-Regular.ttf
   │  ├── CommitMono-700-Regular.ttf
   │  ├── CommitMono-400-Italic.ttf
   │  └── CommitMono-700-Italic.ttf
   ├── installation.txt
   ├── license.txt
   └── CommitMono-700-Italic.otf
```

</details>